notebook/templates/tree.html

```diff
@@ -1,211 +1,252 @@
-{% extends "page.html" %}
```

Comment:
Github is showing that every line in this file has changed, which makes it hard to see what's really changed. I think you've probably changed the line ending style. Can you work out how to change it back to what it was before?